WARNING 2018-04-01 00:26:10,237 base 6820 139652157306624 Not Found: /robots.txt
WARNING 2018-04-01 00:26:10,244 basehttp 6820 139652157306624 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-01 00:26:10,557 base 6820 139652182484736 Not Found: /favicon.ico
WARNING 2018-04-01 00:26:10,557 basehttp 6820 139652182484736 "GET /favicon.ico HTTP/1.1" 404 2334
ERROR 2018-04-01 00:32:10,857 exception 7074 140397864322816 Internal Server Error: /login/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 158, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = response.render()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 83, in rendered_content
    content = template.render(context, self._request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 447, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/urls/base.py", line 88, in reverse
    return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/urls/resolvers.py", line 632, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'cluster' not found. 'cluster' is not a valid view function or pattern name.
ERROR 2018-04-01 00:32:10,973 basehttp 7074 140397864322816 "GET /login/?next=/ HTTP/1.1" 500 176837
ERROR 2018-04-01 00:32:26,105 exception 7074 140397872715520 Internal Server Error: /login/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 158, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = response.render()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 83, in rendered_content
    content = template.render(context, self._request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 447, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/urls/base.py", line 88, in reverse
    return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/urls/resolvers.py", line 632, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'cluster' not found. 'cluster' is not a valid view function or pattern name.
ERROR 2018-04-01 00:32:26,172 basehttp 7074 140397872715520 "GET /login/?next=/ HTTP/1.1" 500 176837
WARNING 2018-04-01 00:33:54,674 basehttp 7074 140397872715520 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:33:58,766 basehttp 7074 140397640673024 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:35:15,348 basehttp 7074 140397872715520 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:35:16,432 basehttp 7074 140397649065728 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:36:06,398 basehttp 7074 140397872715520 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:38:35,895 base 7074 140397623887616 Not Found: /robots.txt
WARNING 2018-04-01 00:38:35,896 basehttp 7074 140397623887616 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-01 00:38:36,197 basehttp 7074 140397864322816 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
ERROR 2018-04-01 00:39:03,642 exception 10016 140537337468672 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 15, in index
    return render(request,'index.html',{
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 488, in parse
    self.unclosed_block_tag(parse_until)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 549, in unclosed_block_tag
    raise self.error(token, msg)
django.template.exceptions.TemplateSyntaxError: Unclosed tag on line 6: 'block'. Looking for one of: endblock.
ERROR 2018-04-01 00:39:03,743 basehttp 10016 140537337468672 "GET / HTTP/1.1" 500 152867
WARNING 2018-04-01 00:41:01,820 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:17,170 base 10016 140537325840128 Not Found: /robots.txt
WARNING 2018-04-01 00:49:17,171 basehttp 10016 140537325840128 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-01 00:49:17,500 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:38,444 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:39,588 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:48,004 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:55,365 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:56,221 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:56,434 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:49:57,589 basehttp 10016 140537325840128 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:50:13,183 basehttp 10016 140537325840128 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:51:03,544 basehttp 10016 140537325840128 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:51:04,436 basehttp 10016 140537325840128 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:51:30,231 basehttp 10016 140537325840128 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:51:30,888 basehttp 10016 140537325840128 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:51:32,065 basehttp 10016 140537092892416 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:51:42,698 basehttp 10016 140537337468672 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:52:24,622 basehttp 13204 139645851846400 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:52:25,319 basehttp 13204 139645851846400 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:52:26,719 basehttp 13204 139645841004288 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:52:27,512 basehttp 13204 139645618743040 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:52:51,202 basehttp 13323 139803254245120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:52:52,146 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:53:33,286 basehttp 13323 139803220412160 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:57:16,681 basehttp 13323 139803228804864 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:57:17,600 basehttp 13323 139803245590272 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:57:30,920 basehttp 13323 139803228804864 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:57:50,244 base 13323 139803209570048 Not Found: /robots.txt
WARNING 2018-04-01 00:57:50,245 basehttp 13323 139803209570048 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-01 00:57:50,494 basehttp 13323 139803254245120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:58:37,655 basehttp 13323 139803228804864 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 00:59:45,250 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:04,286 base 13323 139803254245120 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:04,288 base 13323 139803245590272 Not Found: /DataTables/datatables.min.css
WARNING 2018-04-01 01:07:04,289 basehttp 13323 139803245590272 "GET /DataTables/datatables.min.css HTTP/1.1" 404 2388
WARNING 2018-04-01 01:07:04,289 basehttp 13323 139803254245120 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:04,517 basehttp 13323 139803220412160 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:05,660 base 13323 139803237197568 Not Found: /DataTables/datatables.min.css
WARNING 2018-04-01 01:07:05,661 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.css HTTP/1.1" 404 2388
WARNING 2018-04-01 01:07:05,669 base 13323 139803262637824 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:05,670 basehttp 13323 139803262637824 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:05,897 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:05,938 base 13323 139803262637824 Not Found: /DataTables/datatables.min.css
WARNING 2018-04-01 01:07:05,939 basehttp 13323 139803262637824 "GET /DataTables/datatables.min.css HTTP/1.1" 404 2388
WARNING 2018-04-01 01:07:05,944 base 13323 139803237197568 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:05,945 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:06,112 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:09,422 base 13323 139803237197568 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:09,423 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:22,400 base 13323 139803262637824 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:22,401 base 13323 139803237197568 Not Found: /DataTables/datatables.min.css
WARNING 2018-04-01 01:07:22,402 basehttp 13323 139803262637824 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:22,402 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.css HTTP/1.1" 404 2388
WARNING 2018-04-01 01:07:22,624 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:23,347 base 13323 139803237197568 Not Found: /DataTables/datatables.min.css
WARNING 2018-04-01 01:07:23,348 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.css HTTP/1.1" 404 2388
WARNING 2018-04-01 01:07:23,362 base 13323 139803262637824 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:23,362 basehttp 13323 139803262637824 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:23,608 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:27,516 base 13323 139803237197568 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:27,516 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:30,325 base 13323 139803237197568 Not Found: /DataTables/datatables.min.css
WARNING 2018-04-01 01:07:30,327 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.css HTTP/1.1" 404 2388
WARNING 2018-04-01 01:07:30,337 base 13323 139803237197568 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:30,337 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:30,621 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:31,041 base 13323 139803237197568 Not Found: /DataTables/datatables.min.css
WARNING 2018-04-01 01:07:31,042 basehttp 13323 139803237197568 "GET /DataTables/datatables.min.css HTTP/1.1" 404 2388
WARNING 2018-04-01 01:07:31,048 base 13323 139803262637824 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:31,048 basehttp 13323 139803262637824 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:07:31,353 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:07:35,470 base 13323 139803262637824 Not Found: /DataTables/datatables.min.js
WARNING 2018-04-01 01:07:35,470 basehttp 13323 139803262637824 "GET /DataTables/datatables.min.js HTTP/1.1" 404 2385
WARNING 2018-04-01 01:09:58,529 basehttp 13323 139803237197568 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:09:58,530 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:09:58,734 basehttp 13323 139803262637824 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:09:59,935 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:09:59,941 basehttp 13323 139803237197568 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:10:00,070 basehttp 13323 139803237197568 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:10:24,615 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:10:24,628 basehttp 13323 139803262637824 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:10:24,757 basehttp 13323 139803262637824 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:10:27,764 basehttp 13323 139803262637824 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:10:27,870 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:10:27,871 basehttp 13323 139803237197568 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:10:36,171 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:10:36,173 basehttp 13323 139803262637824 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:10:36,330 basehttp 13323 139803262637824 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:10:46,449 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:10:46,455 basehttp 13323 139803237197568 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:10:46,607 basehttp 13323 139803237197568 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:11:08,807 basehttp 13323 139803262637824 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:11:08,809 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:11:08,960 basehttp 13323 139803237197568 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:13:19,566 base 13323 139803262637824 Not Found: /robots.txt
WARNING 2018-04-01 01:13:19,568 basehttp 13323 139803262637824 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-01 01:13:19,922 basehttp 13323 139803237197568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:13:19,924 basehttp 13323 139803262637824 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:13:20,010 basehttp 13323 139803262637824 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:13:22,199 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:13:22,203 basehttp 13323 139803237197568 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:13:22,320 basehttp 13323 139803237197568 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:13:23,499 basehttp 13323 139803220412160 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:13:23,501 basehttp 13323 139803254245120 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:13:23,634 basehttp 13323 139803254245120 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:13:50,279 basehttp 13323 139803254245120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:13:50,280 basehttp 13323 139803220412160 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:13:50,736 basehttp 13323 139803220412160 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:15:28,664 basehttp 13323 139803220412160 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:15:28,668 basehttp 13323 139803254245120 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:15:29,092 basehttp 13323 139803254245120 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:16:29,895 basehttp 13323 139803209570048 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:16:29,895 basehttp 13323 139803254245120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:16:30,333 basehttp 13323 139803254245120 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:16:37,065 basehttp 13323 139803262637824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:16:37,070 basehttp 13323 139803228804864 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:16:37,471 basehttp 13323 139803228804864 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:17:53,615 basehttp 13323 139803228804864 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 01:17:53,617 basehttp 13323 139803262637824 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 01:17:54,164 basehttp 13323 139803262637824 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 01:21:24,384 base 13323 139803228804864 Not Found: /robots.txt
WARNING 2018-04-01 01:21:24,386 basehttp 13323 139803228804864 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-01 01:22:51,278 base 13323 139803262637824 Not Found: /favicon.ico
WARNING 2018-04-01 01:22:51,279 basehttp 13323 139803262637824 "GET /favicon.ico HTTP/1.1" 404 2334
WARNING 2018-04-01 01:25:11,484 base 13323 139803228542720 Not Found: /favicon.ico
WARNING 2018-04-01 01:25:11,485 basehttp 13323 139803228542720 "GET /favicon.ico HTTP/1.1" 404 2334
WARNING 2018-04-01 01:28:19,766 base 13323 139803262637824 Not Found: /robots.txt
WARNING 2018-04-01 01:28:19,768 basehttp 13323 139803262637824 "GET /robots.txt HTTP/1.1" 404 2331
ERROR 2018-04-01 01:34:19,123 exception 23162 140164509390592 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 17, in index
    all = c.getall()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/aruta/mongodb.py", line 14, in getall
    return db.aruta_export.find({})
NameError: name 'db' is not defined
ERROR 2018-04-01 01:34:19,177 basehttp 23162 140164509390592 "GET / HTTP/1.1" 500 85425
ERROR 2018-04-01 01:34:59,544 exception 23379 139687049406208 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 17, in index
    all = c.getall()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/aruta/mongodb.py", line 14, in getall
    return self.db.aruta_export.find({}).all()
AttributeError: 'Cursor' object has no attribute 'all'
ERROR 2018-04-01 01:34:59,599 basehttp 23379 139687049406208 "GET / HTTP/1.1" 500 85565
ERROR 2018-04-01 01:35:00,451 exception 23379 139687049406208 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 17, in index
    all = c.getall()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/aruta/mongodb.py", line 14, in getall
    return self.db.aruta_export.find({}).all()
AttributeError: 'Cursor' object has no attribute 'all'
ERROR 2018-04-01 01:35:00,492 basehttp 23379 139687049406208 "GET / HTTP/1.1" 500 85565
ERROR 2018-04-01 01:46:56,453 exception 26008 140209060845312 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 23, in index
    if elem.Prepagata not in prepagata.keys():
AttributeError: 'dict' object has no attribute 'Prepagata'
ERROR 2018-04-01 01:46:56,498 basehttp 26008 140209060845312 "GET / HTTP/1.1" 500 83483
ERROR 2018-04-01 01:47:54,386 exception 26008 140209060845312 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 23, in index
    if elem.Prepagata not in prepagata.keys():
AttributeError: 'dict' object has no attribute 'Prepagata'
ERROR 2018-04-01 01:47:54,415 basehttp 26008 140209060845312 "GET / HTTP/1.1" 500 83483
ERROR 2018-04-01 01:47:55,486 exception 26008 140209060845312 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 23, in index
    if elem.Prepagata not in prepagata.keys():
AttributeError: 'dict' object has no attribute 'Prepagata'
ERROR 2018-04-01 01:47:55,516 basehttp 26008 140209060845312 "GET / HTTP/1.1" 500 83483
ERROR 2018-04-01 01:48:18,423 exception 26804 139950749542144 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 23, in index
    print(elem.Prepagata)
AttributeError: 'dict' object has no attribute 'Prepagata'
ERROR 2018-04-01 01:48:18,484 basehttp 26804 139950749542144 "GET / HTTP/1.1" 500 83487
WARNING 2018-04-01 01:50:07,076 base 27272 140693389436672 Not Found: /robots.txt
WARNING 2018-04-01 01:50:07,077 basehttp 27272 140693389436672 "GET /robots.txt HTTP/1.1" 404 2331
ERROR 2018-04-01 01:50:07,082 exception 27272 140693397829376 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 29, in index
    prepagata[elem['Prepagata']][elem['StorageLevel']]['size']+=elem.Size
AttributeError: 'dict' object has no attribute 'Size'
ERROR 2018-04-01 01:50:07,127 basehttp 27272 140693397829376 "GET / HTTP/1.1" 500 83905
ERROR 2018-04-01 01:51:02,036 exception 27505 139743538968320 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 29, in index
    prepagata[elem['Prepagata']][elem['StorageLevel']]['size']+=elem['Size']
TypeError: unsupported operand type(s) for +=: 'int' and 'str'
ERROR 2018-04-01 01:51:02,085 basehttp 27505 139743538968320 "GET / HTTP/1.1" 500 83986
ERROR 2018-04-01 01:51:03,328 exception 27505 139743538968320 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 29, in index
    prepagata[elem['Prepagata']][elem['StorageLevel']]['size']+=elem['Size']
TypeError: unsupported operand type(s) for +=: 'int' and 'str'
ERROR 2018-04-01 01:51:03,355 basehttp 27505 139743538968320 "GET / HTTP/1.1" 500 83986
ERROR 2018-04-01 01:51:23,603 exception 27619 139862757254912 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 30, in index
    prepagata[elem['Prepagata']][elem['StorageLevel']]['size']+=int(elem['Size'])
ValueError: invalid literal for int() with base 10: ''
ERROR 2018-04-01 01:51:23,648 basehttp 27619 139862757254912 "GET / HTTP/1.1" 500 83985
ERROR 2018-04-01 01:51:24,580 exception 27619 139862757254912 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 30, in index
    prepagata[elem['Prepagata']][elem['StorageLevel']]['size']+=int(elem['Size'])
ValueError: invalid literal for int() with base 10: ''
ERROR 2018-04-01 01:51:24,608 basehttp 27619 139862757254912 "GET / HTTP/1.1" 500 83985
ERROR 2018-04-01 01:51:45,627 exception 27724 140169364813568 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 45, in index
    'prepagata':prepagata,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 814, in do_for
    sequence = parser.compile_filter(bits[in_index + 1])
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 568, in compile_filter
    return FilterExpression(token, self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 668, in __init__
    "from '%s'" % (token[upto:], token))
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '()' from 'v.items()'
ERROR 2018-04-01 01:51:45,701 basehttp 27724 140169364813568 "GET / HTTP/1.1" 500 221398
ERROR 2018-04-01 01:51:46,496 exception 27724 140169364813568 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 45, in index
    'prepagata':prepagata,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 814, in do_for
    sequence = parser.compile_filter(bits[in_index + 1])
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 568, in compile_filter
    return FilterExpression(token, self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 668, in __init__
    "from '%s'" % (token[upto:], token))
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '()' from 'v.items()'
ERROR 2018-04-01 01:51:46,555 basehttp 27724 140169364813568 "GET / HTTP/1.1" 500 221398
WARNING 2018-04-01 01:59:15,874 base 29246 140139043313408 Not Found: /favicon.ico
WARNING 2018-04-01 01:59:15,875 basehttp 29246 140139043313408 "GET /favicon.ico HTTP/1.1" 404 2334
WARNING 2018-04-01 02:09:10,487 base 30836 140483250480896 Not Found: /robots.txt
WARNING 2018-04-01 02:09:10,489 basehttp 30836 140483250480896 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-01 02:12:47,391 base 30836 140483267528448 Not Found: /favicon.ico
WARNING 2018-04-01 02:12:47,391 basehttp 30836 140483267528448 "GET /favicon.ico HTTP/1.1" 404 2334
WARNING 2018-04-01 02:14:03,693 base 30836 140483267528448 Not Found: /favicon.ico
WARNING 2018-04-01 02:14:03,693 basehttp 30836 140483267528448 "GET /favicon.ico HTTP/1.1" 404 2334
WARNING 2018-04-01 02:14:08,076 basehttp 30836 140483267528448 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:14:08,091 basehttp 30836 140483267528448 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:14:08,091 basehttp 30836 140483258873600 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:14:36,198 basehttp 30836 140483238852352 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:14:36,201 basehttp 30836 140483250480896 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:14:36,621 basehttp 30836 140483267528448 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:14:59,833 basehttp 30836 140483267528448 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:14:59,836 basehttp 30836 140483238852352 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:15:00,232 basehttp 30836 140483230459648 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:15:31,600 basehttp 30836 140483230459648 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:15:31,601 basehttp 30836 140483267528448 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:15:31,989 basehttp 30836 140483250480896 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:16:13,057 basehttp 30836 140483250480896 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:16:13,058 basehttp 30836 140483230459648 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:16:13,496 basehttp 30836 140483238852352 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:18:25,179 basehttp 30836 140483250480896 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:18:25,179 basehttp 30836 140483238852352 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:18:25,635 basehttp 30836 140483267528448 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:19:19,638 basehttp 30836 140483250480896 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:19:19,642 basehttp 30836 140483258873600 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:19:20,220 basehttp 30836 140483230459648 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:21:08,833 basehttp 30836 140483230459648 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:21:08,835 basehttp 30836 140483267528448 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:21:09,280 basehttp 30836 140483238852352 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-01 02:21:41,125 basehttp 30836 140483238852352 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-01 02:21:41,126 basehttp 30836 140483230459648 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-01 02:21:41,557 basehttp 30836 140483258873600 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 09:48:22,580 base 31162 140691327117056 Not Found: /robots.txt
WARNING 2018-04-02 09:48:22,582 basehttp 31162 140691327117056 "GET /robots.txt HTTP/1.1" 404 2331
ERROR 2018-04-02 09:48:22,595 exception 31162 140691369080576 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 475, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 73, in index
    'utente':utente,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 477, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 533, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 101: 'static', expected 'endblock'. Did you forget to register or load this tag?
ERROR 2018-04-02 09:48:22,706 basehttp 31162 140691369080576 "GET / HTTP/1.1" 500 206337
WARNING 2018-04-02 10:06:21,676 basehttp 1377 140635742922496 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:06:22,242 basehttp 1377 140635751315200 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:06:22,244 basehttp 1377 140635726137088 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:06:31,276 basehttp 1377 140635734529792 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:06:31,285 basehttp 1377 140635970733824 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:06:31,844 basehttp 1377 140635751315200 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:07:27,308 basehttp 1377 140635742922496 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:07:27,308 basehttp 1377 140635751315200 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:07:27,884 basehttp 1377 140635962341120 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:07:57,973 basehttp 1377 140635962341120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:07:57,973 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:07:58,545 basehttp 1377 140635726137088 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:09:45,466 basehttp 1377 140635726137088 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:09:45,467 basehttp 1377 140635717744384 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:09:45,949 basehttp 1377 140635742922496 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:10:38,471 base 1377 140635717744384 Not Found: /robots.txt
WARNING 2018-04-02 10:10:38,474 basehttp 1377 140635717744384 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-02 10:10:38,798 basehttp 1377 140635970733824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:10:38,799 basehttp 1377 140635742922496 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:10:39,394 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:10:57,875 basehttp 1377 140635726137088 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:10:57,876 basehttp 1377 140635751315200 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:10:58,304 basehttp 1377 140635709351680 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:12:01,052 basehttp 1377 140635698509568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:12:01,057 basehttp 1377 140635709351680 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:12:01,480 basehttp 1377 140635962341120 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:13:13,251 basehttp 1377 140635751315200 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:13:13,252 basehttp 1377 140635962341120 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:13:13,855 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:13:37,840 basehttp 1377 140635970733824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:13:37,840 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:13:38,422 basehttp 1377 140635717482240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:13:52,033 basehttp 1377 140635717482240 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:13:52,034 basehttp 1377 140635751315200 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:13:52,512 basehttp 1377 140635742922496 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:15:03,881 basehttp 1377 140635698509568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:15:03,881 basehttp 1377 140635742922496 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:15:04,417 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:16:11,822 basehttp 1377 140635962341120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:16:11,822 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:16:12,225 basehttp 1377 140635726137088 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:16:19,521 basehttp 1377 140635742922496 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:16:19,521 basehttp 1377 140635726137088 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:16:20,052 basehttp 1377 140635717482240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:16:43,925 basehttp 1377 140635698509568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:16:43,927 basehttp 1377 140635717482240 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:16:44,460 basehttp 1377 140635962341120 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:16:58,194 basehttp 1377 140635962341120 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:16:58,195 basehttp 1377 140635970733824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:16:58,613 basehttp 1377 140635742922496 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:17:09,425 basehttp 1377 140635726137088 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:17:09,426 basehttp 1377 140635742922496 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:17:09,834 basehttp 1377 140635717482240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:17:18,034 basehttp 1377 140635717482240 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:17:18,037 basehttp 1377 140635970733824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:17:18,428 basehttp 1377 140635962341120 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:17:32,896 basehttp 1377 140635742922496 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:17:32,897 basehttp 1377 140635962341120 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:17:33,280 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:17:39,479 basehttp 1377 140635970733824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:17:39,482 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:17:39,943 basehttp 1377 140635717482240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:20:26,656 basehttp 1377 140635717482240 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:20:26,656 basehttp 1377 140635698509568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:20:27,094 basehttp 1377 140635751315200 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:20:45,994 basehttp 1377 140635970733824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:20:45,997 basehttp 1377 140635751315200 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:20:48,607 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:20:59,507 base 1377 140635698509568 Not Found: /robots.txt
WARNING 2018-04-02 10:20:59,508 basehttp 1377 140635698509568 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-02 10:20:59,790 basehttp 1377 140635751315200 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:20:59,791 basehttp 1377 140635970733824 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:21:00,235 basehttp 1377 140635742922496 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:21:37,088 basehttp 1377 140635214444288 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:21:37,093 basehttp 1377 140635717482240 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:21:37,488 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:22:34,620 basehttp 1377 140635970733824 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:22:34,623 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:22:35,053 basehttp 1377 140635751315200 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:23:21,353 basehttp 1377 140635742922496 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:23:21,354 basehttp 1377 140635751315200 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:23:21,771 basehttp 1377 140635717482240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:27:31,949 basehttp 1377 140635962341120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:27:31,953 basehttp 1377 140635717482240 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:27:32,372 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:28:03,597 basehttp 1377 140635751315200 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:28:03,605 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:28:04,036 basehttp 1377 140635726137088 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:28:17,194 basehttp 1377 140635726137088 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:28:17,195 basehttp 1377 140635962341120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:28:18,302 basehttp 1377 140635717482240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:29:44,413 basehttp 1377 140635717482240 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:29:44,414 basehttp 1377 140635742922496 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:29:44,929 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:29:51,816 basehttp 1377 140635962341120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:29:51,818 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:29:52,256 basehttp 1377 140635726137088 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:30:41,051 basehttp 1377 140635698509568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:30:41,051 basehttp 1377 140635726137088 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:30:41,466 basehttp 1377 140635751315200 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:31:31,645 basehttp 1377 140635962341120 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:31:31,651 basehttp 1377 140635751315200 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:31:32,048 basehttp 1377 140635734529792 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:31:35,409 basehttp 1377 140635726137088 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:31:35,410 basehttp 1377 140635734529792 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:31:37,600 basehttp 1377 140635717482240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:33:34,726 basehttp 10043 139667114194688 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:33:34,726 basehttp 10043 139666900645632 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:33:35,143 basehttp 10043 139666892252928 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:34:12,528 basehttp 10043 139666900645632 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:34:12,536 basehttp 10043 139666900645632 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:34:12,536 basehttp 10043 139666858682112 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:34:50,425 basehttp 10043 139666867074816 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:34:50,429 basehttp 10043 139666900645632 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:34:50,854 basehttp 10043 139666883860224 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:35:58,216 basehttp 10043 139667122587392 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:35:58,216 basehttp 10043 139666883860224 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:35:58,641 basehttp 10043 139666892252928 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:36:01,476 basehttp 10043 139666883860224 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:36:01,480 basehttp 10043 139666858682112 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:36:01,866 basehttp 10043 139667122587392 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:36:53,924 basehttp 10043 139666883860224 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:36:53,924 basehttp 10043 139667114194688 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:36:54,342 basehttp 10043 139666892252928 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:36:56,860 basehttp 10043 139666900645632 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:36:56,863 basehttp 10043 139666892252928 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:36:57,281 basehttp 10043 139667122587392 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:37:03,553 basehttp 10043 139666883860224 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:37:03,556 basehttp 10043 139667114194688 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:37:03,929 basehttp 10043 139666875467520 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:37:12,459 basehttp 10043 139666875467520 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:37:12,461 basehttp 10043 139666867074816 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:37:12,907 basehttp 10043 139666892252928 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:38:04,198 base 10043 139666883860224 Not Found: /robots.txt
WARNING 2018-04-02 10:38:04,199 basehttp 10043 139666883860224 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-02 10:38:04,562 basehttp 10043 139666892252928 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:38:04,562 basehttp 10043 139667114194688 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:38:05,178 basehttp 10043 139666883860224 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:38:11,567 basehttp 10043 139666875467520 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:38:11,570 basehttp 10043 139666867074816 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:38:11,848 basehttp 10043 139667114194688 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:38:11,849 basehttp 10043 139667122587392 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:38:12,222 basehttp 10043 139666883860224 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:39:49,704 basehttp 10043 139666867074816 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:39:49,704 basehttp 10043 139666875467520 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:39:50,105 basehttp 10043 139667122587392 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:40:16,971 basehttp 10043 139667122587392 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:40:16,976 basehttp 10043 139667114194688 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:40:17,406 basehttp 10043 139666892252928 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:40:33,097 basehttp 10043 139666883860224 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:40:33,097 basehttp 10043 139666875467520 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:40:33,513 basehttp 10043 139666867074816 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:40:49,967 basehttp 10043 139666858682112 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:40:49,967 basehttp 10043 139667114194688 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:40:50,386 basehttp 10043 139666883860224 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 10:41:50,890 basehttp 10043 139667114194688 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 10:41:50,895 basehttp 10043 139666867074816 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 10:41:51,458 basehttp 10043 139666892252928 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
ERROR 2018-04-02 10:43:59,550 exception 10043 139667114194688 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/smartif.py", line 175, in translate_token
    op = OPERATORS[token]
KeyError: "'size and s != 'price"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 77, in index
    'utente':utente,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 955, in do_if
    condition = TemplateIfParser(parser, bits).parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 888, in __init__
    super().__init__(*args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/smartif.py", line 166, in __init__
    mapped_tokens.append(self.translate_token(token))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/smartif.py", line 177, in translate_token
    return self.create_var(token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 891, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 568, in compile_filter
    return FilterExpression(token, self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 668, in __init__
    "from '%s'" % (token[upto:], token))
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: 'price' from ''size and s != 'price'
ERROR 2018-04-02 10:43:59,691 basehttp 10043 139667114194688 "GET / HTTP/1.1" 500 273001
ERROR 2018-04-02 10:46:19,919 exception 13074 140553861076736 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 767, in __init__
    self.literal = float(var)
ValueError: could not convert string to float: 'v._price_'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 788, in __init__
    self.literal = mark_safe(unescape_string_literal(var))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/utils/functional.py", line 196, in wrapper
    return func(*args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/utils/text.py", line 397, in unescape_string_literal
    raise ValueError("Not a string literal: %r" % s)
ValueError: Not a string literal: 'v._price_'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 77, in index
    'utente':utente,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 454, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 452, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 568, in compile_filter
    return FilterExpression(token, self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 653, in __init__
    var_obj = Variable(var)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 795, in __init__
    var)
django.template.exceptions.TemplateSyntaxError: Variables and attributes may not begin with underscores: 'v._price_'
ERROR 2018-04-02 10:46:20,033 basehttp 13074 140553861076736 "GET / HTTP/1.1" 500 252485
ERROR 2018-04-02 10:46:20,761 exception 13074 140553852684032 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 767, in __init__
    self.literal = float(var)
ValueError: could not convert string to float: 'v._price_'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 788, in __init__
    self.literal = mark_safe(unescape_string_literal(var))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/utils/functional.py", line 196, in wrapper
    return func(*args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/utils/text.py", line 397, in unescape_string_literal
    raise ValueError("Not a string literal: %r" % s)
ValueError: Not a string literal: 'v._price_'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 77, in index
    'utente':utente,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 454, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 452, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 568, in compile_filter
    return FilterExpression(token, self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 653, in __init__
    var_obj = Variable(var)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 795, in __init__
    var)
django.template.exceptions.TemplateSyntaxError: Variables and attributes may not begin with underscores: 'v._price_'
ERROR 2018-04-02 10:46:20,853 basehttp 13074 140553852684032 "GET / HTTP/1.1" 500 252485
WARNING 2018-04-02 10:53:50,073 base 15108 140279438534400 Not Found: /robots.txt
WARNING 2018-04-02 10:53:50,075 basehttp 15108 140279438534400 "GET /robots.txt HTTP/1.1" 404 2331
WARNING 2018-04-02 11:31:06,769 base 23245 140388416362240 Not Found: /dettaglio/
WARNING 2018-04-02 11:31:06,769 basehttp 23245 140388416362240 "GET /dettaglio/ HTTP/1.1" 404 1696
WARNING 2018-04-02 11:31:41,604 base 23245 140388416362240 Not Found: /dettaglio/
WARNING 2018-04-02 11:31:41,604 basehttp 23245 140388416362240 "GET /dettaglio/ HTTP/1.1" 404 1696
WARNING 2018-04-02 11:31:45,465 base 23245 140388425017088 Not Found: /favicon.ico
WARNING 2018-04-02 11:31:45,466 basehttp 23245 140388425017088 "GET /favicon.ico HTTP/1.1" 404 2464
WARNING 2018-04-02 11:32:18,647 base 23559 140682238588672 Not Found: /robots.txt
WARNING 2018-04-02 11:32:18,650 basehttp 23559 140682238588672 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-02 11:32:21,375 base 23559 140682229933824 Not Found: /dettaglio/
WARNING 2018-04-02 11:32:21,375 basehttp 23559 140682229933824 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 404 1717
ERROR 2018-04-02 11:34:08,605 exception 23977 140178037307136 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 96, in dettaglio
    find = c.findPrepagata()
TypeError: findPrepagata() missing 2 required positional arguments: 'p' and 's'
ERROR 2018-04-02 11:34:08,649 basehttp 23977 140178037307136 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 500 77369
ERROR 2018-04-02 11:34:27,498 exception 24045 140578161792768 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 96, in dettaglio
    find = c.findPrepagata(request.GET.get('p'),request.GET.get('s'))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/aruta/mongodb.py", line 18, in findPrepagata
    regex = re.compile(["^", s, "$"].join(""),re.IGNORECASE);
AttributeError: 'list' object has no attribute 'join'
ERROR 2018-04-02 11:34:27,559 basehttp 24045 140578161792768 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 500 81716
ERROR 2018-04-02 11:34:57,790 exception 24158 139855667570432 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 105, in dettaglio
    'prepagata':prepagata,
NameError: name 'prepagata' is not defined
ERROR 2018-04-02 11:34:57,833 basehttp 24158 139855667570432 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 500 76931
ERROR 2018-04-02 11:35:25,248 exception 24286 139746527389440 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 106, in dettaglio
    'prepagata':prepagata,
NameError: name 'prepagata' is not defined
ERROR 2018-04-02 11:35:25,297 basehttp 24286 139746527389440 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 500 76885
ERROR 2018-04-02 11:35:58,947 exception 24391 140357652182784 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 106, in dettaglio
    'prepagata':prepagata,
NameError: name 'prepagata' is not defined
ERROR 2018-04-02 11:35:58,988 basehttp 24391 140357652182784 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 500 76885
ERROR 2018-04-02 11:36:54,756 exception 24575 139817828480768 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 98, in dettaglio
    'find':find
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: dettaglio.html
ERROR 2018-04-02 11:36:54,803 basehttp 24575 139817828480768 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 500 92412
ERROR 2018-04-02 11:39:13,613 exception 24799 140067637581568 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 101, in dettaglio
    'prepagata':prepagata,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: dettaglio.html
ERROR 2018-04-02 11:39:13,660 basehttp 24799 140067637581568 "GET /dettaglio/?p=DATABASE&s=All HTTP/1.1" 500 92583
WARNING 2018-04-02 11:39:18,509 basehttp 24799 140067406345984 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:39:18,560 basehttp 24799 140067406345984 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:39:18,566 basehttp 24799 140067397953280 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:39:20,073 basehttp 24799 140067389560576 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:39:20,075 basehttp 24799 140067397953280 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:39:20,548 basehttp 24799 140067406345984 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:39:48,924 basehttp 24799 140067406345984 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:39:48,924 basehttp 24799 140067389560576 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:39:49,409 basehttp 24799 140067637581568 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:39:54,959 basehttp 24799 140067389560576 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:39:54,959 basehttp 24799 140067397953280 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:39:55,357 basehttp 24799 140067406345984 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:43:06,264 basehttp 26034 140331059873536 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:43:06,265 basehttp 26034 140331051480832 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:43:06,683 basehttp 26034 140331043088128 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:43:09,779 basehttp 26034 140331026302720 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:43:09,781 basehttp 26034 140331043088128 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:43:10,116 basehttp 26034 140330666026752 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:43:53,292 basehttp 26034 140330674419456 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:43:53,293 basehttp 26034 140330666026752 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:43:53,650 basehttp 26034 140331068266240 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:44:09,877 basehttp 26034 140331068266240 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:44:09,881 basehttp 26034 140331034695424 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:44:10,252 basehttp 26034 140330674419456 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:54:12,588 basehttp 27237 140337972500224 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:54:12,589 basehttp 27237 140337980892928 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:54:12,991 basehttp 27237 140337964107520 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:54:17,892 basehttp 27237 140337947322112 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:54:17,894 basehttp 27237 140337955714816 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:54:18,189 basehttp 27237 140337955714816 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:54:33,177 basehttp 27237 140337938929408 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:54:33,177 basehttp 27237 140337964107520 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:54:33,506 basehttp 27237 140337964107520 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:02,952 basehttp 27237 140337980892928 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:02,954 basehttp 27237 140337964107520 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:03,344 basehttp 27237 140337964107520 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:13,362 basehttp 27237 140337938929408 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:13,365 basehttp 27237 140337955714816 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:13,665 basehttp 27237 140337955714816 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:24,440 base 27237 140337964107520 Not Found: /robots.txt
WARNING 2018-04-02 11:55:24,441 basehttp 27237 140337964107520 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-02 11:55:24,690 basehttp 27237 140337964107520 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:24,695 basehttp 27237 140337955714816 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:25,116 basehttp 27237 140337955714816 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:33,378 basehttp 27237 140337938929408 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:33,379 basehttp 27237 140337964107520 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:33,839 basehttp 27237 140337964107520 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:35,035 basehttp 27237 140337972500224 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:35,046 basehttp 27237 140337972500224 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:35,049 basehttp 27237 140337955714816 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:35,839 basehttp 27237 140337955714816 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:35,846 basehttp 27237 140337435629312 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:35,846 basehttp 27237 140337955714816 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:36,354 basehttp 27237 140337418843904 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:36,358 basehttp 27237 140337955714816 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:36,746 basehttp 27237 140337972500224 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:55:38,057 base 27237 140337972500224 Not Found: /dettaglio/{% url
WARNING 2018-04-02 11:55:38,057 basehttp 27237 140337972500224 "GET /dettaglio/%7B%%20url HTTP/1.1" 404 2485
WARNING 2018-04-02 11:55:40,380 basehttp 27237 140337972500224 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:55:40,383 basehttp 27237 140337964107520 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:55:40,830 basehttp 27237 140337964107520 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:56:32,674 basehttp 27237 140337964107520 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:56:32,678 basehttp 27237 140337418843904 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:56:33,126 basehttp 27237 140337418843904 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:57:04,432 basehttp 27237 140337418843904 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:57:04,436 basehttp 27237 140337402058496 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:57:04,794 basehttp 27237 140337972500224 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:57:07,623 basehttp 27237 140337972500224 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:57:07,623 basehttp 27237 140337418843904 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:57:07,936 basehttp 27237 140337955714816 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:57:14,830 basehttp 27237 140337955714816 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:57:14,834 basehttp 27237 140337402058496 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:57:15,136 basehttp 27237 140337402058496 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 11:59:10,599 basehttp 29874 140653193225984 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 11:59:10,603 basehttp 29874 140653165336320 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 11:59:10,936 basehttp 29874 140653165336320 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 13:58:31,801 basehttp 20239 139875407886080 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 13:58:31,804 basehttp 20239 139875380258560 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 13:58:32,380 basehttp 20239 139875399493376 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 13:58:57,026 basehttp 20239 139875407886080 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 13:58:57,027 basehttp 20239 139875399493376 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 13:58:57,555 basehttp 20239 139875399493376 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 14:01:15,260 basehttp 20239 139875399493376 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 14:01:15,261 basehttp 20239 139875363473152 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 14:01:15,731 basehttp 20239 139875380258560 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 14:05:50,470 base 20239 139874980054784 Not Found: /robots.txt
WARNING 2018-04-02 14:05:50,470 basehttp 20239 139874980054784 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-02 14:05:53,101 base 20239 139874996840192 Not Found: /dettaglio/
WARNING 2018-04-02 14:05:53,102 basehttp 20239 139874996840192 "GET /dettaglio/?u=U385361&s= HTTP/1.1" 404 1713
WARNING 2018-04-02 14:07:13,703 basehttp 22558 140321144960768 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 14:07:13,703 basehttp 22558 140321109202688 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 14:09:11,492 basehttp 22558 140321117595392 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 14:09:11,493 basehttp 22558 140321109202688 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 14:09:11,936 basehttp 22558 140321144960768 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 14:09:18,034 basehttp 22558 140321144960768 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 14:09:18,040 basehttp 22558 140321117595392 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 14:09:18,509 basehttp 22558 140321117595392 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
ERROR 2018-04-02 14:12:47,466 exception 23896 139695253903104 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 127, in dettaglio
    size += int(elem['Size'])
ValueError: invalid literal for int() with base 10: ''
ERROR 2018-04-02 14:12:47,530 basehttp 23896 139695253903104 "GET /dettaglio/?u=U385361&s=All HTTP/1.1" 500 79733
ERROR 2018-04-02 14:13:15,905 exception 24378 139819036440320 Internal Server Error: /dettaglio/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 141, in dettaglio
    'price':price,
UnboundLocalError: local variable 'price' referenced before assignment
ERROR 2018-04-02 14:13:15,970 basehttp 24378 139819036440320 "GET /dettaglio/?u=U385361&s=All HTTP/1.1" 500 80977
WARNING 2018-04-02 14:13:28,366 basehttp 24425 140414924109568 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 14:13:28,366 basehttp 24425 140414951999232 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 14:13:28,936 basehttp 24425 140414924109568 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 14:13:50,873 basehttp 24425 140414924109568 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 14:13:50,874 basehttp 24425 140414907324160 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 14:13:51,420 basehttp 24425 140414907324160 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 14:14:00,933 basehttp 24425 140414951999232 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-02 14:14:00,938 basehttp 24425 140414915716864 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-02 14:14:01,595 basehttp 24425 140414924109568 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-02 14:36:49,837 base 29508 140640516044544 Not Found: /robots.txt
WARNING 2018-04-02 14:36:49,839 basehttp 29508 140640516044544 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-02 14:36:53,207 base 29508 140640516044544 Not Found: /favicon.ico
WARNING 2018-04-02 14:36:53,208 basehttp 29508 140640516044544 "GET /favicon.ico HTTP/1.1" 404 2464
WARNING 2018-04-02 14:37:50,014 base 29801 140288562763520 Not Found: /robots.txt
WARNING 2018-04-02 14:37:50,018 basehttp 29801 140288562763520 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-03 14:15:13,777 base 12345 140374041327360 Not Found: /robots.txt
WARNING 2018-04-03 14:15:13,780 basehttp 12345 140374041327360 "GET /robots.txt HTTP/1.1" 404 2461
ERROR 2018-04-03 14:15:13,820 exception 12345 140374091683584 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 82, in index
    'storage_level': StorageLevel.objects.all()
NameError: name 'StorageLevel' is not defined
ERROR 2018-04-03 14:15:14,253 basehttp 12345 140374091683584 "GET / HTTP/1.1" 500 89560
WARNING 2018-04-03 14:25:11,892 base 12540 139903363344128 Not Found: /robots.txt
WARNING 2018-04-03 14:25:11,915 basehttp 12540 139903363344128 "GET /robots.txt HTTP/1.1" 404 2461
ERROR 2018-04-03 14:25:19,470 exception 12540 139903392282368 Internal Server Error: /admin/app/costostorage/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 158, in __get__
    rel_obj = self.field.get_cached_value(instance)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/mixins.py", line 13, in get_cached_value
    return instance._state.fields_cache[cache_name]
KeyError: 'storage_level'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 158, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = response.render()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 83, in rendered_content
    content = template.render(context, self._request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/library.py", line 214, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    'results': list(results(cl))}
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 302, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 293, in __init__
    super().__init__(*items)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 212, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/utils.py", line 295, in lookup_field
    value = getattr(obj, name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 164, in __get__
    rel_obj = self.get_object(instance)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 139, in get_object
    return qs.get(self.field.get_reverse_related_filter(instance))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/query.py", line 394, in get
    clone = self.filter(*args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/query.py", line 836, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/query.py", line 854, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1253, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1271, in _add_q
    current_negated, allow_joins, split_subq)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1277, in _add_q
    split_subq=split_subq,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1215, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1085, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/lookups.py", line 18, in __init__
    self.rhs = self.get_prep_lookup()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/lookups.py", line 68, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 947, in get_prep_value
    return int(value)
ValueError: invalid literal for int() with base 10: 'Silver'
ERROR 2018-04-03 14:25:19,643 basehttp 12540 139903392282368 "GET /admin/app/costostorage/ HTTP/1.1" 500 343723
ERROR 2018-04-03 14:27:24,770 exception 15592 140636347012864 Internal Server Error: /admin/app/costostorage/
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 158, in __get__
    rel_obj = self.field.get_cached_value(instance)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/mixins.py", line 13, in get_cached_value
    return instance._state.fields_cache[cache_name]
KeyError: 'storage_level'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 158, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 156, in _get_response
    response = response.render()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/response.py", line 83, in rendered_content
    content = template.render(context, self._request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/library.py", line 214, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    'results': list(results(cl))}
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 302, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 293, in __init__
    super().__init__(*items)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/templatetags/admin_list.py", line 212, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/admin/utils.py", line 295, in lookup_field
    value = getattr(obj, name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 164, in __get__
    rel_obj = self.get_object(instance)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 139, in get_object
    return qs.get(self.field.get_reverse_related_filter(instance))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/query.py", line 394, in get
    clone = self.filter(*args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/query.py", line 836, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/query.py", line 854, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1253, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1271, in _add_q
    current_negated, allow_joins, split_subq)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1277, in _add_q
    split_subq=split_subq,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1215, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1085, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/lookups.py", line 18, in __init__
    self.rhs = self.get_prep_lookup()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/lookups.py", line 68, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 947, in get_prep_value
    return int(value)
ValueError: invalid literal for int() with base 10: 'Silver'
ERROR 2018-04-03 14:27:24,893 basehttp 15592 140636347012864 "GET /admin/app/costostorage/ HTTP/1.1" 500 351890
ERROR 2018-04-03 14:30:59,587 exception 15881 140294100088576 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 475, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 82, in index
    'storage_level': StorageLevel.objects.all()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 198, in compile_nodelist
    return parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 483, in parse
    raise self.error(token, e)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/defaulttags.py", line 956, in do_if
    nodelist = parser.parse(('elif', 'else', 'endif'))
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 477, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/template/base.py", line 533, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 90: 'endfor', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
ERROR 2018-04-03 14:30:59,775 basehttp 15881 140294100088576 "GET / HTTP/1.1" 500 271317
WARNING 2018-04-03 14:32:33,888 base 15881 140294074910464 Not Found: /robots.txt
WARNING 2018-04-03 14:32:33,892 basehttp 15881 140294074910464 "GET /robots.txt HTTP/1.1" 404 2461
ERROR 2018-04-03 14:44:35,696 exception 20206 139706796193536 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 47, in index
    prepagata[elem['Prepagata']][l_sl]={'price_':0,'size_':0}
TypeError: unhashable type: 'list'
ERROR 2018-04-03 14:44:35,758 basehttp 20206 139706796193536 "GET / HTTP/1.1" 500 78855
ERROR 2018-04-03 14:44:48,777 exception 20301 139648457828096 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 75, in index
    prepagata[elem['Utente']][sl]={'price_':0,'size_':0}
KeyError: 'U385361'
ERROR 2018-04-03 14:44:48,830 basehttp 20301 139648457828096 "GET / HTTP/1.1" 500 79138
WARNING 2018-04-03 14:59:57,990 base 21190 139869091260160 Not Found: /robots.txt
WARNING 2018-04-03 14:59:57,996 basehttp 21190 139869091260160 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-03 15:01:42,042 basehttp 21190 139869082867456 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:01:42,096 basehttp 21190 139869074474752 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:01:42,101 basehttp 21190 139869082867456 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:05:18,300 basehttp 21190 139869074474752 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:05:18,313 basehttp 21190 139869099652864 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:05:19,228 basehttp 21190 139869099652864 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:06:12,523 basehttp 21190 139869074474752 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:06:12,533 basehttp 21190 139869082867456 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:06:13,576 basehttp 21190 139869082867456 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:07:09,738 basehttp 21190 139869074474752 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:07:09,742 basehttp 21190 139869099652864 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:07:10,537 basehttp 21190 139869099652864 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:07:30,108 basehttp 21190 139869082867456 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:07:30,109 basehttp 21190 139869074474752 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:07:30,826 basehttp 21190 139869074474752 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:09:17,258 base 21190 139869082867456 Not Found: /robots.txt
WARNING 2018-04-03 15:09:17,260 basehttp 21190 139869082867456 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-03 15:09:17,934 basehttp 21190 139869099652864 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:09:17,935 basehttp 21190 139869091260160 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:09:18,700 basehttp 21190 139869091260160 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:39:31,468 basehttp 21190 139869099652864 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:39:31,472 basehttp 21190 139869074474752 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:39:31,473 basehttp 21190 139869091260160 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:41:08,232 base 21190 139869099652864 Not Found: /robots.txt
WARNING 2018-04-03 15:41:08,257 basehttp 21190 139869099652864 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-03 15:41:09,632 basehttp 21190 139869099652864 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:41:09,635 basehttp 21190 139869091260160 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:41:10,778 basehttp 21190 139869091260160 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:42:33,417 basehttp 21190 139869082867456 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:42:33,419 basehttp 21190 139869091260160 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:42:34,370 basehttp 21190 139869099652864 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:42:38,634 basehttp 21190 139869074474752 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:42:38,634 basehttp 21190 139869091260160 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:42:39,245 basehttp 21190 139869091260160 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
WARNING 2018-04-03 15:43:38,982 basehttp 21190 139869091260160 "GET /static/css/bootstrap.min.css.map HTTP/1.1" 404 1699
WARNING 2018-04-03 15:43:38,986 basehttp 21190 139869082867456 "GET /static/DataTables/bootstrap.min.css.map HTTP/1.1" 404 1720
WARNING 2018-04-03 15:43:39,781 basehttp 21190 139869074474752 "GET /static/DataTables/pdfmake.min.js.map HTTP/1.1" 404 1711
ERROR 2018-04-03 15:52:04,081 exception 5016 140182650836736 Internal Server Error: /
Traceback (most recent call last):
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/env/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/media/vittorio/eddc0c27-3d32-4ad5-8799-c62fa3bb805f/Lavori/QUADRATA/aruta/src/aruta/app/views.py", line 32, in index
    for elem in Servizio.objects.all():
NameError: name 'Servizio' is not defined
ERROR 2018-04-03 15:52:04,189 basehttp 5016 140182650836736 "GET / HTTP/1.1" 500 77471
WARNING 2018-04-03 15:55:22,309 base 6265 140565389854464 Not Found: /robots.txt
WARNING 2018-04-03 15:55:22,310 basehttp 6265 140565389854464 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-03 17:23:37,304 base 24634 140480125859584 Not Found: /robots.txt
WARNING 2018-04-03 17:23:37,308 basehttp 24634 140480125859584 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-03 17:24:46,358 base 24804 140087697606400 Not Found: /robots.txt
WARNING 2018-04-03 17:24:46,360 basehttp 24804 140087697606400 "GET /robots.txt HTTP/1.1" 404 2461
WARNING 2018-04-03 17:26:56,891 base 25073 140093968516864 Not Found: /favicon.ico
WARNING 2018-04-03 17:26:56,892 basehttp 25073 140093968516864 "GET /favicon.ico HTTP/1.1" 404 2464
WARNING 2018-04-03 17:29:02,890 base 25073 140093951731456 Not Found: /robots.txt
WARNING 2018-04-03 17:29:02,895 basehttp 25073 140093951731456 "GET /robots.txt HTTP/1.1" 404 2461
